sdcm/kcl_thread.py

```diff
@@ -0,0 +1,136 @@
+import os
+import time
+import random
+import logging
+import uuid
+import threading
+
+from sdcm.stress_thread import format_stress_cmd_error, DockerBasedStressThread
+from sdcm.utils.docker_remote import RemoteDocker
+from sdcm.sct_events import StressEvent, InfoEvent, Severity
+from sdcm.cluster import BaseNode
+
+LOGGER = logging.getLogger(__name__)
+
+
+class KclStressEvent(StressEvent):
+    pass
+
+
+class KclStressThread(DockerBasedStressThread):  # pylint: disable=too-many-instance-attributes
+
+    def run(self):
+        _self = super().run()
+        # wait for the KCL thread to create the tables, so the YCSB thread beat this one, and start failing
```

Comment:
not sure it is clear to understand - why is it needed for ycsb thread to fail? why does it take 3 minutes?

Reply:
YCSB needed the table to be created.
The KclThread code is the part creating the tables, and configure then with streams enable.

If YCSB starts before hydra-kcl it might fail, since tables are not yet created